### ISSUE TITLE
loading EMM386.EXE in CONFIG.SYS causes Dosbox-X to reboot

### ISSUE THREAD

**Issue by Siddhant-Mashalkar** [joncampbell123/dosbox-x#2742]
i install MS-DOS 7.1 in Dosbox-X on a hdd image , i booted into the hard disk image , and dosbox-x rebooted quickly
i had to remove the DEVICE=EMM386.EXE line from config.sys to fix
why?

**Comment by Wengier** [comment]
I think this issue is similar to issue https://github.com/joncampbell123/dosbox-x/issues/2661, that booting images with EMM386 may cause incompatibility issues at this time.

**Comment by Siddhant-Mashalkar** [comment]
okay , how do i fix the rebooting of dosbox-x , should i replace EMM386 with Jemm386 or Jemmex? or something else?

**Comment by Wengier** [comment]
Have you tried to replace it with JEMM386/JEMMEX and see if it works?

**Comment by Siddhant-Mashalkar** [comment]
replacing emm386 with jemm386 worked for me
![image](https://user-images.githubusercontent.com/71384075/127609824-737863b4-2f1e-4ef5-9155-cc3eb0e47f55.png)

**Comment by Wengier** [comment]
There is now JEMM386 5.80, and you can try the latest version too. Also, in OP you said you tried to install MS-DOS 7.1 but the screenshot shows 6.22. You may want to try MS-DOS 7.1 too for this.

**Comment by Siddhant-Mashalkar** [comment]
yeah , i installed DOS 6.22 instead , now it works, GSCDROM.SYS (CD-ROM Driver) and RAMDRIVE.SYS (ramdisk driver) also works
![image](https://user-images.githubusercontent.com/71384075/127610583-a74c5262-3de3-4578-83b3-b96b0576c3a1.png)

**Comment by Siddhant-Mashalkar** [comment]
> I think this issue is similar to issue #2661, that booting images with EMM386 may cause incompatibility issues at this time.

@Wengier i tried Mini-Windows 3.11 Boot Disk, if i start Mini-Windows 3.11 with EMM386, dosbox-x quickly reboots
how can i fix this? do i need to change Dosbox-X Config file?
